charts/csi-powermax/values.yaml

```diff
@@ -101,7 +130,7 @@ defaultFsType: ext4
 #  IfNotPresent: Only pull the image if it does not already exist on the node.
 #  Never: Never pull the image.
 # Default value: None
-imagePullPolicy: IfNotPresent
```

Comment:
Default value of IfNotPresent should remain for customers so they don't always pull this image

Reply:
changed